README.md

```diff
@@ -68,7 +67,32 @@ class AccountListView(LoginRequiredMixin, SmartListMixin, ListView):
     ]
 ```
 
-### License
+## Development
```

Comment:
Cool stuff. Just fyi If you want to test the changes in your project you can do something like that in your docker compose in the container that uses smart lists: 
```
    volumes:
      - /Users/michalsimonienko/projects/django-smart-lists/smart_lists:/venv/lib/python2.7/site-packages/smart_lists
``